stubs/model.py

```diff
@@ -2044,8 +2064,35 @@ def dolfin_set_function_values(self, sp, ukey, unew):
         elif isinstance(unew, (float, int)):
             uinterp = d.interpolate(d.Constant(unew), sp.V)
             d.assign(sp.u[ukey], uinterp)
+        elif len(unew) > 1:
```

Comment:
So the idea here is:
You have a function space `V` on a parent mesh (and function `u`), and a set of child-meshes with functions `uc_i` in compartments (presumably of the same topological dimension, i.e. no facet meshes), and you want to fill `u` such that `u = sum_{i} interpolate(uc_i)`?
or am I missing something?